azurerm/internal/services/subscription/subscription_resource_test.go

```diff
@@ -56,22 +56,31 @@ func TestAccSubscriptionResource_update(t *testing.T) {
 	if os.Getenv("ARM_BILLING_ACCOUNT") == "" {
 		t.Skip("skipping tests - no billing account data provided")
 	}
+
 	data := acceptance.BuildTestData(t, "azurerm_subscription", "test")
 	r := SubscriptionResource{}
+	assert := check.That(data.ResourceName)
 
 	data.ResourceTest(t, r, []acceptance.TestStep{
 		{
 			Config: r.basicEnrollmentAccount(data),
 			Check: acceptance.ComposeTestCheckFunc(
-				check.That(data.ResourceName).ExistsInAzure(r)),
+				assert.ExistsInAzure(r),
+				assert.Key("tags.%").HasValue("2"),
+				assert.Key("tags.cost_center").HasValue("MSFT"),
+				assert.Key("tags.environment").HasValue("Production"),
```

Comment:
Can we keep these in long-hand form as with the rest of the test suite?

Reply:
Looks like this one was missed from the last review? Can we get these in the form:
```suggestion
				check.That(data.ResourceName).ExistsInAzure(r),
				check.That(data.ResourceName).Key("tags.%").HasValue("2"),
				check.That(data.ResourceName).Key("tags.cost_center").HasValue("MSFT"),
				check.That(data.ResourceName).Key("tags.environment").HasValue("Production"),
```
The consistency helps us make bulk changes if/when we update approaches to the test framework.